main.go

```diff
@@ -185,6 +186,13 @@ func addParserDefaultOptions(options *GlobalOptions) {
 		// automatically normalize the request when using the nginx parser
 		options.RequestShape = append(options.RequestShape, "request")
 	}
+	if options.Reqs.ParserName != "mysql" {
+		// mysql is the only parser that requires post-parsed sampling.
+		// Sample all other parser when tailing to conserve CPU
+		options.TailSample = true
+	} else {
+		options.TailSample = false
```

Comment:
won't this just clobber anything that gets passed in? is the intent for the flag to be purely a global switch, rather than an actual settable flag?